src/mockServiceWorker.js

```diff
@@ -112,7 +112,7 @@ self.addEventListener('fetch', function (event) {
   }
 
   // Generate unique request ID.
-  const requestId = Math.random().toString(16).slice(2)
+  const requestId = crypto.randomUUID().slice(0, 8)
```

Comment:
Can you please tell me more about why we need to `.slice()` the generated UUID? 🤔

Reply:
Ah, good catch, for some weird reason I thought the previous command outputted 8 chars and I tried to replicate the same behaviour - which is actually incorrect as `Math.random().toString(16).slice(2)` results in 13 characters 😅 

Anyways, I'll remove the `slice` and leave the original output (which is 36 characters long) 😬